odilia/src/tower.rs

```diff
@@ -224,7 +455,7 @@ where
 							match cmds.send(ok).await {
 								Ok(()) => {}
 								Err(e) => {
-									tracing::error!("Could not send command {:?} over channel! This usually means either the channel is full, which is bad!", e);
+									tracing::error!("Could not send command {:?} over channel! This usually means the channel is full, which is bad!", e);
```

Comment:
It is a bounded channrl.

Reply:
why? can't it be unbounded? what could the user do in case their logs are spammed with that, presumably serious error in the logs? or maybe I shouldn't think of logs for the users, should I? haha

Reply:
The logs are primarily for us.

That said, I'm mostly trying to somewhat limit memory usage. If the channel gets big, there a lot of allod/dealloc that has to happen to clear it out. Maybe I'm paranoid about this.